### PR TITLE
[HomeComponent] Added home component

### PR DESCRIPTION
**Pull request by SriramPandikasala**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,14 +1,11 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
 import './App.css'
+import { HomeComponent } from './module/HomeComponent/HomeComponent'
 
 function App() {
-  const [count, setCount] = useState(0)
```
Comment (VidyasagarPandikashala) [inline src/App.tsx:7]: do not remove this

File: src/App.tsx [hunk 2]
```diff
@@ -1,14 +1,11 @@
...
 
   return (
     <>
-    This is React
+       <HomeComponent></HomeComponent>
```
Comment (VidyasagarPandikashala) [inline src/App.tsx]: - please create a root component and add home component under it, let the root component handle the home component
 - `<RootComponent/>`

File: src/App.tsx (hunk 1 above)
Comment (SriramPandikasala) [reply src/App.tsx:7]: As discussed,  we are not using this. So will be keeping it removed

File: src/App.tsx (hunk 2 above)
Comment (SriramPandikasala) [reply src/App.tsx]: Will do this

**Review by VidyasagarPandikashala — APPROVED** [review]

**Pull request merged**